Dockerfile.exporter

```
@@ -1,12 +1,12 @@
 ARG BUILDPLATFORM=linux/amd64
-FROM --platform=$BUILDPLATFORM debian:bullseye-slim as importer
+FROM --platform=$BUILDPLATFORM golang:1.19.2-bullseye AS builder
 
 ARG PROMETHEUS_VARNISH_EXPORTER_VERSION=1.6.1
```

declare them again bc ARGs are not shared between stages.

```suggestion
ARG TARGETOS
ARG TARGETARCH
ARG PROMETHEUS_VARNISH_EXPORTER_VERSION=1.6.1
```